protos/flyteidl/service/admin.proto

```diff
@@ -248,6 +248,16 @@ service AdminService {
     };
   }
 
+  rpc RecoverExecution (flyteidl.admin.ExecutionRecoverRequest) returns (flyteidl.admin.ExecutionCreateResponse) {
+    option (google.api.http) = {
+      post: "/api/v1/executions/recover"
+      body: "*"
+    };
+    option (grpc.gateway.protoc_gen_swagger.options.openapiv2_operation) = {
+      description: "Recovers a previously-run workflow execution that failed due to system failure."
```

Comment:
can we improve the description - 
In Recover mode, users cannot change any input parameters or update the version of the execution. This is extremely useful to recover from system errors and byzantine faults like - Loss of K8s cluster, bugs in platform or instability, machine failures, downstream system failures (downstream services), or simply to recover executions that failed because of retry exhaustion and should complete if tried again.

Reply:
also add a `:ref:`?

Reply:
also add about transient failures

Reply:
so the generated rst docs all pick up the comments above, these descriptions are only used for swagger. I can make them consistent-ish although the rst links don't work.